ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3391,6 +3381,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-long-duration
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones
+- as: vsphere-ipi-zones-multisubnets-external-lb-f28
```

Comment:
Based on https://github.com/openshift/release/pull/42996 is adding destructive jobs, to keep consistent, pls add one more destructive job for it.

Reply:
ok